domene/src/main/java/no/nav/foreldrepenger/fpformidling/tilkjentytelse/TilkjentYtelseForeldrepenger.java

```diff
@@ -31,6 +31,10 @@ public boolean equals(Object o) {
         if (o == null || getClass() != o.getClass())
             return false;
         TilkjentYtelseForeldrepenger that = (TilkjentYtelseForeldrepenger) o;
+        if (perioder != null && that.perioder != null) {
```

Comment:
Den er OK hvis listen ikke kan være null. Ellers må du gjøre null sjekk slikt:
       
   ```
 private boolean erListeLik(List<Periode> l1, List< Periode > l2) {
        if (l1 == null && l2 == null)
            return true;
        if (l1 == null || l2 == null)
            return false;
        return l1.size() == l2.size() && l2.containsAll(l1);
    }
```

Reply:
Fanges av Objects.equals(perioder, that.perioder); under